astropy/utils/xml/iterparser.py

```diff
@@ -130,7 +130,8 @@ def get_xml_iterator(source, _debug_python_based_parser=False):
 
     Parameters
     ----------
-    fd : readable file-like object or read function
+    source : readable file-like or callable
+        Function to read the data.
```

Comment:
Should this be "Handle that contains the data or function that reads it"?
(Also elsewhere in this file)

Reply:
✅ & stole some more description from ``_convert_to_fd_or_read_function``, which actually handles the input.